app/scripts/lib/auth-errors.js

```diff
@@ -279,6 +287,14 @@ define(function (require, exports, module) {
           return {
             param: err.param
           };
+        } else if (this.is(err, 'INVALID_RESUME_TOKEN_PROPERTY')) {
+          return {
+            property: err.property
```

Comment:
Happy to do it that way: I did consider it but decided that `property` was more accurate than `param` in this case, given the name and nature of the error. Want me to change it anyway?